### ISSUE TITLE
The example depends on a deprecated version of pkg_tar

### ISSUE THREAD

**Issue by aiuto** [google/rpmpack#74]
example_bazel/BULID.bazel contains the line
`load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar").

That should be changed to `load("@rules_pkg//pkg:tar.bzl", "pkg_tar")`
and rules_license added to the WORKSPACE as per https://github.com/bazelbuild/rules_pkg/releases/tag/0.7.0

In Bazel 6.0 your example may cease to work, either because pkg_tar is removed, or simply because the mode and ownername attributes are removed.

**Comment by jarondl** [comment]
Thanks @aiuto for the heads up! Seems like an easy fix, I'll do that.

**Comment by jarondl** [comment]
Oh boy, I had some fun here. I kept getting errors using `load("@rules_pkg//pkg:tar.bzl", "pkg_tar")`, it kept saying `pkg` is not a package. Using `load("@rules_pkg//:pkg.bzl", "pkg_tar")` worked, but that clearly means I'm using an old version of rules_pkg. Looking into the cached directories, I found the reason: `io_bazel_rules_docker` also calls rules_pkg, but with the 0.2.6 version.

Aliasing my import to rules_pkg_070 fixed it, but seems ugly. I am really surprised bazel let me shadow a version of a dependency with the same name.
